types/errors/errors.go

```diff
@@ -135,6 +135,10 @@ var (
 	// supported.
 	ErrNotSupported = Register(RootCodespace, 37, "feature not supported")
 
+	// ErrIO should be used to wrap internal errors caused by external operation.
+	// Examples: not DB domain error, file writing etc...
+	ErrIO = Register(RootCodespace, 38, "Internal IO error")
```

Comment:
it seems to me no-one is ever throwing this error. Would it make sense to introduce it when we need it?

Reply:
I'm using it for differentiating an internal keyring error, from other errors.